sabnzbd/directunpacker.py

```diff
@@ -78,6 +78,15 @@ def save(self):
         pass
 
     def reset_active(self):
+        # make sure the process and filehandles are closed nicely:
+        try:
+            # Creation was done via "self.active_instance = Popen()", so:
+            self.active_instance.stdout.close()
+            self.active_instance.stdin.close()
+            self.active_instance.wait(timeout=2)
+        except:
+            logging.debug("Exception in reset_active()")
```

Comment:
I would suggest to include the traceback:
```python
logging.debug("Exception in reset_active()", exc_info=True)
```

Reply:
OK, I'll do.